doc/source/development.rst

```diff
@@ -45,14 +45,15 @@ for details). For Python 2.7 use :command:`virtualenv`, which is provided
 via pip or as an extra package in your favourite Linux distribution.
 
 However, for setting up a Python virtual development environment the
-following additional include, header files and compilers are required
+following additional latex, include, header files and compilers are required
```

Comment:
You are probably referring to the project (LaTeX) instead of the command (latex), right? If yes, replace latex -> LaTeX. :wink: